bee-network/bee-autopeering/src/config.rs

```diff
@@ -96,6 +99,11 @@ impl AutopeeringConfig {
         self.entry_nodes
     }
 
+    /// The peer storage path.
+    pub fn peer_storage_path(&self) -> &Path {
```

Comment:
why `peer_storage_path` and not `peer_store_path`?

Reply:
`peer_store_path` sounds odd to me -- that could be the path that peers use to store something.

Reply:
To be consistent with the other storage that shares the same root. And store is weird, it really is a storage.

Reply:
@Adam-Gleave well, it used to be `peerstore` but someone (:wink: ) wanted the underscore. Would `peerstore_path` be okay?

Reply:
That wouldn't be rust nor TOML compliant